docs/concepts/mixer.md

```diff
@@ -7,40 +7,210 @@ title: Mixer
 {% endcapture %}
 
 {% capture body %}
-## General usage
+## Background
 
 The mixer provides the control-plane abstractions necessary for most real-world multi-tenant services.
 The proxy delegates policy decisions to the mixer and dispatches its telemetry data to the mixer, which
 proceeds to repackage and redirect the data towards configured backends.
 
-Services within the Istio mesh can also directly integrate with the mixer. For example, services may wish to provide rich telemetry for particular operations
-beyond what the proxy automatically collects. Or services may use the mixer for resource-oriented quota management. Services that leverage the mixer in this 
-way are abstracted from environment-specific control plane details, greatly easing the process of hosting the code in
-different environments (different clouds & on-prem)
+Services within the Istio mesh can also directly integrate with the mixer. For example, services may wish to provide rich telemetry
+for particular operations beyond what the proxy automatically collects. Or services may use the mixer for resource-oriented quota
+management. Services that leverage the mixer in this way are abstracted from environment-specific control plane details, greatly
+easing the process of  hosting the code in different environments (different clouds & on-prem)
+
+<img style="display:block;width:60%;margin:auto;" src="./mixer/ingress.svg" alt="Flow of inbound traffic to a service." />
+<p style="text-align:center;">Ingress Traffic</p><br/>
+
+<img style="display:block;width:60%;margin:auto;" src="./mixer/egress.svg" alt="Flow of outbound traffic to a service." /> 
+<p style="text-align:center;">Egress Traffic</p>
 
 The mixer provides three core features:
 
-**Precondition Checking**. Enables callers to verify a number of preconditions before responding to an incoming request from a service consumer. 
+- **Precondition Checking**. Enables callers to verify a number of preconditions before responding to an incoming request from a service consumer. 
 Preconditions can include whether the service consumer is properly authenticated, is on the service's whitelist, passes ACL checks, and more.
 
-**Telemetry Reporting**. Enables services to produce logging, monitoring, tracing and billing streams intended for the service producer itself as well as 
+- **Telemetry Reporting**. Enables services to produce logging, monitoring, tracing and billing streams intended for the service producer itself as well as 
 for its consumers.
 
-**Quota Management**. Enables services to allocate and free quota on a number of dimensions, Quotas are used as a relatively simple resource management 
+- **Quota Management**. Enables services to allocate and free quota on a number of dimensions, Quotas are used as a relatively simple resource management 
 tool to provide some fairness between service consumers when contending for limited resources.
 
+## Attributes
```

Comment:
i feel like this should link to a more detailed Istio-level discussion of attributes. Attributes are used by mixer for sure, but the proxy and/or mixerclient bits should also talk about them.

Also, I think we should link to existing docs on well-known attributes somewhere.